blog/tests/admin/test_post_delete_service.py

```
@@ -8,6 +8,16 @@
 from blog.services.post_delete_service import delete_post
 
 
+@pytest.fixture(autouse=True)
+def aws_mock():
+    with mock_aws():
```

Ruff (0.15.6)

[error] Undefined name `mock_aws`

(F821)